### ISSUE TITLE
Font-Size Difference between Linux and Windows

### ISSUE THREAD

**Issue by creharmony** [hajareshyam/pdf-creator-node#54]
Hi
I notice a strong difference about font-size between windows 10 and linux Ubuntu.

To get a quite similar text size I need to apply 
on linux:
```
body { font-size: 8px; }
```
and on windows
```
body { font-size: 12px; }
```

I don't know if this issue is related to pdf-creator-node library but if this is the case, it would be great to retrieve a similar size between different os.

**Comment by jonfrusciante** [comment]
> Hi I notice a strong difference about font-size between windows 10 and linux Ubuntu.
> 
> To get a quite similar text size I need to apply on linux:
> 
> ```
> body { font-size: 8px; }
> ```
> 
> and on windows
> 
> ```
> body { font-size: 12px; }
> ```
> 
> I don't know if this issue is related to pdf-creator-node library but if this is the case, it would be great to retrieve a similar size between different os.

I have also the dome problem

**Comment by sandeepDevJs** [comment]
yes ... i am facing the same issue ... font size seems bigger when works on Linux ubuntu.

**Comment by monterico** [comment]
i was facing the same issue. the solution is in Linux i add CSS code: `html{zoom:0.7}` to reduce the font

**Comment by sandeepDevJs** [comment]
@monterico, thanks for the suggestion I'll try this out.

**Comment by akshay-sood** [comment]
Seems like a common issue with no solution (there than CSS zoom:0.55 or font-size: 50% etc).

Other references: 
https://github.com/marcbachmann/node-html-pdf/issues/421
https://github.com/marcbachmann/node-html-pdf/issues/110
